docs/getting_started/index.rst

```diff
@@ -57,16 +79,35 @@ ctapipe repo (make sure you put in your own username there):
     $ cd ctapipe
 
 
-You now need to tell Git that this repo where the master CTA version is:
+You now need to tell Git that this repo where the main ctapipe repository is:
```

Comment:
Sounds like the _that this repo_ is superflous here.

Reply:
Maybe better: *If you are using a fork, the default remote repository will be the one in your user account on GitHub. To make sure you are in sync with the official copy in the cta-observatory account, you need to add an upstream remote as follows:*

Reply:
That's a good addtion, I adapted this slightly to not imply that by merely adding the remote, things are synchronized.